enterprise/frontend/src/embedding-sdk/tests/styling-sdk-tests.stories.tsx

```diff
@@ -11,7 +11,7 @@ export default {
 
 const configThatWillError: SDKConfig = {
   apiKey: "TEST",
-  metabaseInstanceUrl: "http://localhost",
+  metabaseInstanceUrl: "http://fake-host:1234",
```

Comment:
I run again into the issue that I had something listening on localhost:80 which made this test to fail so I changed this to make it less likely that's there a server listening there until we address https://github.com/metabase/metabase/issues/48145